src/libexpr/tests/yaml.cc

```diff
@@ -0,0 +1,161 @@
+#ifdef HAVE_RYML
+
+#include "libexpr.hh"
+
+// Ugly, however direct access to the SAX parser is required in order to parse multiple JSON objects from a stream
+#include "json-to-value.cc"
+
+
+namespace nix {
+// Testing the conversion from YAML
+
+    static std::string replaceUnicodePlaceholders(std::string_view str) {
```

Comment:
What is `replaceUnicodePlaceholders()` needed for? Maybe add a comment to clarify.

Reply:
The test files contain some Unicode characters to indicate the presence/absence of some characters like spaces, tabs and newlines: https://github.com/yaml/yaml-test-suite/tree/main#special-characters
`replaceUnicodePlaceholders()` replaces these characters.